compile/customWrapper/pyRunner.m

```diff
@@ -0,0 +1,39 @@
+function [output,sub_rough] = pyRunner(filename, functionName, args)
+    % Calls a Python custom function from the base Matlab
+    % workspace. The arguments should be full or relative path of the 
+    % python file with extension, the name of the function to call in the 
+    % python file and a cell array of argument for the reflectivity calculation
+    %
+    % [o, s] = pythonWrapper.runner('layers.py', 'layer', {params, bulk_in, bulk_out, contrast});
+    % [o, s] = pythonWrapper.runner('layers.py', 'layer', {params, bulk_in, bulk_out, contrast, domain});
+    
+    [filepath, name, ~] = fileparts(filename);
+    insert(py.sys.path, int32(0), filepath)
+    
+    module = py.importlib.import_module(name);
+    py.importlib.reload(module);       
+    
+    % Call the python function
+    if (numel(args) == 4)
+        out = module.(functionName)(args{1}, args{2}, args{3}, int32(args{4} - 1));
+    else
+        out = module.(functionName)(args{1}, args{2}, args{3}, int32(args{4} - 1), int32(args{5} - 1));
+    end
```

Comment:
The signatures are fixed as far as I know. Generalizing will involve a similar if statement and maybe a for loop and no guarantee it will work for the magnetic or any other future signature so I will leave as is for now